src/main/java/uk/ac/cam/cl/dtg/teaching/pottery/model/RepoInfo.java

```diff
@@ -36,21 +38,29 @@
   /** If this value is set then indicates that this repo is hosted remotely. */
   private String remote;
 
+  private int mutation;
```

Comment:
what's this field for?

Reply:
It's slightly an onion now. It seems to make sense to record it (or the seed) but it's not actually used any more as everything is stored in the parameters.

Reply:
Also gives an easy way to report the task mutation back to a client of Pottery in a compact form (e.g. PA), for statistical purposes.